mappings/net/minecraft/player/stats/StatTracker.mapping

```diff
@@ -0,0 +1,15 @@
+CLASS net/minecraft/class_48 net/minecraft/player/stats/StatTracker
+	FIELD field_1271 statMap Ljava/util/Map;
+	FIELD field_1273 statSyncer Lnet/minecraft/class_47;
+	METHOD <init> (Lnet/minecraft/class_49;Ljava/io/File;)V
+		ARG 1 session
+		ARG 2 saveFolder
+	METHOD method_1046 incrementStat (Lnet/minecraft/class_339;I)V
+		ARG 1 stat
+		ARG 2 ammount
+	METHOD method_1048 (Ljava/lang/String;Ljava/lang/String;Ljava/util/Map;)Ljava/lang/String;
+		ARG 1 sessionId
+	METHOD method_1050 increaseStat (Ljava/util/Map;Lnet/minecraft/class_339;I)V
+		ARG 1 statMap
+		ARG 2 stat
+		ARG 3 ammount
```

Comment:
ammount -> amount. I think mojang also called it stat syncher and not syncer? not sure.